sources/reference/admiral-aws-marketplace.md

```diff
@@ -7,100 +7,120 @@ page_keywords: install, microservices, Continuous Integration, Continuous Deploy
+not to renew, upon completing the annual term, billing will automatically
+convert to hourly billing.
+
+Follow the prompts within the AWS Marketplace purchase process to launch your
+Shippable Server SE instance and proceed to the Admin panel.
 
 ## Admin panel
 
-Shippable Server Admin panel can be accessed from a local browser using
+The Shippable Server Admin panel can be accessed from a local browser using
 http-over-ssh. This ensures that the admin panel is only available to the users
 who have ssh permissions for the server.
 
 ### Steps to access Admin panel
-- If the IP address of the server is `1.2.3.4`, then run the following command
-  on your local system to access admin panel.
-```
-$ ssh -L 50003:1.2.3.4:50003 -C ubuntu@1.2.3.4
-```
-- This will establish a secure tunnel between the local system and Shippable
-  Server
-- To access the admin page, navigate to `http://localhost:50003` on your
-  browser.
-  - This should prompt for a login token
-  - The login token is the EC2 instance ID of the server. This can be fetched
-    from the AWS EC2 instances panel
+- Assuming the IP address of the EC/2 instance is `1.2.3.4`, run the following
+  command on your local system to establish a secure tunnel between the local
+  system and Shippable Server and access the admin panel:
+
+    ```
+    $ ssh -L 50003:1.2.3.4:50003 -C ubuntu@1.2.3.4
+    ```  
+
+- Navigate to `http://localhost:50003` from your browser
+- Login to the Admin panel with your login token
+    - The login token is the EC2 Instance ID of the server found in the AWS
+      Management Console > EC/2 > Instances, e.g. "i-0968e71046e294b56"
 
 ## Configuration
 
-- After successfully logging in, you must create passwords for
-  `Messaging` and `State`. These should be more than 8 characters to be valid.
+### Initialize Infrastructure
+- After successfully logging in to the Admin panel, you must create passwords
+  for the `Messaging` and `State` components in the `Initialize Infrastructure` 
+  section. These must be 8 characters or more.
 
-- The `Installer Access Key` and `Installer Secret Key` fields can be left
-  blank.
+- The `Installer Access Key` and `Installer Secret Key` fields are not required
+  for the Startup Edition and can be left blank.
 
 - Once passwords are set, click `Initialize`. This will prep the system with
   core dependencies like database, message queue, secrets store, docker etc.
 
+### Configure & Install
 - After initialization is complete, navigate to the next section `Configure and
-  Install` to set up authentication provider(s).
-
-- In the `Authorization and Source Control Management (SCM) Providers`,
-  section, select the SCM provider of choice. We'll configure Github as
-  a provider as an example.
-  - Check the box for `Github`, `Auth` column
-  - On `https://github.com/settings/developers` register a new application
+  Install` to set up your authentication provider(s). Shippable leverages one 
```

Comment:
Either commas or parentheses around `Configure and Install`.  And, is it "and" or "&" in the UI?  You have both here.